kimchi_navigation/src/global_localization.cpp

```diff
@@ -134,9 +133,12 @@ void GlobalLocalizationServer::execute(
       return;
     }
 
+    RotateRobot();
```

Comment:
For safety reasons, you need to continue sending velocity commands to the real robot